Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -29,12 +32,15 @@
     private final DoubleProperty fVelProp;
     private final DoubleProperty leftDriveEncoderTicksProp;
     private final DoubleProperty rightDriveEncoderTicksProp;
-    private final DoubleProperty ticksPerInch;
     private final DoubleProperty startPositionTicks;
     private final DoubleProperty currentDisplacementInchProp;
+    private final DoubleProperty highGearTicksPerInch;
+    private final DoubleProperty lowGearTicksPerInch;
+    
+    private ShiftSubsystem shift;
```

Comment:
where possible, we've been using longer names for subsystem variables, like `shiftSubsystem`.